test/lightning_web/live/project_live_test.exs

```diff
@@ -1311,22 +1312,11 @@ defmodule LightningWeb.ProjectLiveTest do
       end)
     end
 
-    test "all project users can see the project webhook auth methods", %{
-      conn: conn
-    } do
+    test "all project users can see the project webhook auth methods" do
       project = insert(:project)
       auth_methods = insert_list(4, :webhook_auth_method, project: project)
 
-      for project_user <-
-            Enum.map([:editor, :admin, :owner, :viewer], fn role ->
-              insert(:project_user,
-                role: role,
-                project: project,
-                user: build(:user)
-              )
-            end) do
-        conn = log_in_user(conn, project_user.user)
-
+      for conn <- project_user_conns(project, [:editor, :admin, :owner, :viewer]) do
```

Comment:
A little nitpick, could we name this like `build_conn_for_user_roles`?

Or similar, something that implies it builds things

Or split it between roles as the enum, and the build conn function be more generic?